### ISSUE TITLE
A11y_.Net Core_Winforms_MainForm_ConmonControl1_Keyboard : The Keyboard Focus is not clearly Visible in the Checkbox present in the "Date Time Picker Section"

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/winforms#9228]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

Yes

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG2.4.7;#DesktopApp;#Win11;#Benchmark;#keyboard;

### Actual:
The Keyboard Focus is not clearly Visible in the check box present in the "Date Time Picker" Section.

### Expected:
The Keyboard focus should be visible properly on the check box , when the focus lands on it.

### User Impact:
Low Vision user using keyboard to navigate will face difficulty in knowing the keyboard focus.

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Tab till "Common_Controls1:Testing the controls under the Common_controls" and Hit enter.
4. Navigate to Date Time picker section.
5. Observe with keyboard that the keyboard focus is not clearly visible on the check box.

### Attachment:
![Check box focus](https://github.com/dotnet/winforms/assets/85863895/f3782855-735d-4261-a02d-afc289bf3180)

**Comment by Ashutoshsahoo164** [comment]
@Ashley-Li Are you working on this bug to fix it?

**Comment by dmitrii-drobotov** [comment]
It looks like this is related to native DateTimePicker control from Windows, and there is no way to change selected color from WinForms. I checked native DTP in an MFC app and it has the same issue:
![datetimepicker-mfc](https://github.com/dotnet/winforms/assets/102954094/117c7f6e-4ac9-47dd-b7e5-2773e3cd447c)

CC @merriemcgaw @Tanya-Solyanik

**Comment by Tanya-Solyanik** [comment]
@LeafShi1 , @SimonZhao888 , @Epica3055  - please take a look at this bug and investigation done by @dmitrii-drobotov , this is a good representative of external issues.

**Comment by Ashutoshsahoo164** [comment]
Hi @merriemcgaw it under the exception right? is there any exception link i can refer?

**Comment by merriemcgaw** [comment]
I will work with the office hours to get a parent exception approved. Right now we're doing one-offs for every instance where Windows doesn't let us do something. I'd prefer not to have to file a new exception every time. When it's ready I'll provide a link.

**Comment by 12-shweta** [comment]
@merriemcgaw  thanks for above information.
could you please help us with link through which we can track this issue.

**Comment by merriemcgaw** [comment]
The exception has already been filed. 

WinForms is a thin wrapper around Win32 APIs. In this case it's the CommonControls (DateTimePicker). We have multiple exceptions for this code, and any time we file a bug on Windows it's automatically rejected as legacy UI.  I want to get a blanked exception for all CommonControls in the cases where WinForms has no way to impact the situation. Same thing with CommonDialog APIs.

**Comment by merriemcgaw** [comment]
The exception is filed [here](https://ceacc.visualstudio.com/ceaccex/_workitems/edit/165846/) (internal URL) 

This particular issue we can't change, and it's not a very common scenario for our users.

**Comment by merriemcgaw** [comment]
Waiting on approval of [C&AI Exception ](https://ceacc.visualstudio.com/ceaccex/_workitems/edit/165846/)process

**Comment by PoojaNamde** [comment]
#A11ypreview;

Added A11ypreview tag

**Comment by merriemcgaw** [comment]
@PoojaNamde the exception is now approved. I am closing this issue, please add the appropriate tags to the issue for your tracking.

**Comment by kupatkar99** [comment]
#Closed;
